packages/components/src/modal/style.scss

```diff
@@ -5,53 +5,44 @@
 	right: 0;
 	bottom: 0;
 	left: 0;
-	background-color: rgba($black, 0.7);
+	padding: 20px;
+	background: $white;
```

Comment:
I feel like this should be animated as well since it suddenly shows up.

Reply:
The idea was that if we animate the overlay instead, it feels like there's a lot happening. While if we animate just the content it's more subtle.